src/main/java/uk/gov/digital/ho/hocs/casework/api/dto/CreateStageResponse.java

```diff
@@ -15,7 +15,7 @@
     @JsonProperty("uuid")
     private final UUID uuid;
 
-    public static CreateStageResponse from(StageWithCaseData stage) {
+    public static CreateStageResponse from(BaseStage stage) {
```

Comment:
This response literally only needs a UUID, I see why you haven't done it to minimize the diff, but feels silly chucking the whole object around...